common/src/format.rs

```diff
@@ -384,14 +404,20 @@ impl FormatSpec {
 
     fn get_separator_interval(&self) -> usize {
         match self.format_type {
+            Some(FormatType::Number) => 9999,
```

Comment:
Is this meaning something or just intended a big number? I think bigint can have more digits than 9999.